src/data/data1d/Data1DManager.js

```diff
@@ -8,7 +8,7 @@ export class Data1DManager {
   static fromBruker = function fromBruker(result, options = {}) {
     const { info } = options;
     let data = getData(result.spectra);
-    if (data.im) info.isComplex = true;
+    if (!!data.im) info.isComplex = true;
```

Comment:
In which case this is useful ?

Reply:
Currently, I am not sure which is a better solution to set isComplex in data came from brukerConverter. Or do you mean the !!?

Reply:
Yes the '!!' . What is the value of data.im ? Why it was not working without '!!'

Reply:
you are just right